content/authors.tsv

```diff
@@ -1,3 +1,4 @@
 github_username	full_name	initials	orcid	email	affiliations	funders
 dhimmel	Daniel S. Himmelstein	DSH	0000-0002-3012-7446	daniel.himmelstein@gmail.com	Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania	GBMF4552
 agitter	Anthony Gitter	AG	0000-0002-5324-9833	gitter@biostat.wisc.edu	Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison and Morgridge Institute for Research	NIH U54AI117924
+vsmalladi	Venkat S. Malladi	VSM	0000-0002-0144-0564	vsmalladi@gmail.com	The Laboratory of Signaling and Gene Expression, Cecil H. and Ida Green Center for Reproductive Biology Sciences, University of Texas Southwestern Medical Center	
```

Comment:
I don't care either way, but I don't think there is much incentive for other new contributors to add themselves to the `authors.tsv` list.  We should rely on https://github.com/greenelab/manubot-rootstock/graphs/contributors to see who contributed.  I'm even fine with my name being removed as long as we have 2-3 example authors.